generators/app/templates/src/redux/store.js

```diff
@@ -1,25 +1,28 @@
-import { createStore, applyMiddleware, compose, combineReducers } from 'redux';
-import thunk from 'redux-thunk';
-import { routerReducer as router, routerMiddleware } from 'react-router-redux';
-import createHistory from 'history/createBrowserHistory';
+import { connectRouter, routerMiddleware } from 'connected-react-router';
```

Comment:
We might need to add the fetchMiddleware from redux-recompose I think

Reply:
If we are going to use r-r v2.0, we'll also need `configureMergeState` and optionally, `wrapCombineReducers`.

Reply:
Yeah! There is a lot of work about redux in the bootstrap. Since will be a optional feature, a lot of components and integrations should be refactored. But again, doesn't enter into the scope of this pr